muskathlon/models/account_invoice.py

```diff
@@ -29,7 +29,7 @@ def _after_transaction_invoice_paid(self, transaction):
                     self.env.ref('muskathlon.event_type_muskathlon'):
                 registration.confirm_registration()
```

Comment:
You can completely remove this file and all what it's doing. It was confirming the registration after the payment was done. But instead you should move this call to `registration.confirm_registration()` inside the registration form directly when they submit. We will automatically and directly confirm every registrations.